src/commands/config/mod.rs

```diff
@@ -27,21 +27,18 @@ pub fn global_config(user: &GlobalUser, verify: bool) -> Result<(), failure::Err
         validate_credentials(user)?;
     }
 
-    let toml = toml::to_string(&user)?;
-
     let config_dir = get_global_config_dir().expect("could not find global config directory");
     fs::create_dir_all(&config_dir)?;
 
-    let config_file = config_dir.join("default.toml");
-    fs::write(&config_file, &toml)?;
+    let config_path = user.to_file(&config_dir)?;
```

Comment:
by moving this logic into the GlobalUser struct, we keep the name of the file in the same place.